### DISCUSSION TITLE
Training separately for men and women

### DISCUSSION THREAD

**Discussion by gogogoglo** [dayoonkwon/BioAge#2] [Q&A] [answered]
Hi there,

Was wondering if kdm_nhanes and hd_nhanes functions automatically train separately for men and women? Or is it necessary to create two datasets and run separately?

Thanks very much!

**Comment by dayoonkwon** [comment, marked as answer]
"kdm_nhanes" and "hd_nhanes"  functions automatically train algorithms separately for men and women. But when you use "kdm_calc" and "hd_calc" functions to project those measures onto new data, you will need separate training for gender.
